polyply/src/random_walk.py

```diff
@@ -375,7 +395,7 @@ def _random_walk(self, meta_molecule):
 
         vector_bundle = self.vector_sphere.copy()
         count = 0
-        path = list(nx.bfs_edges(meta_molecule, source=first_node))
+        path = list(nx.dfs_edges(meta_molecule, source=first_node))
```

Comment:
@pckroon this is something I'm currently trying to figure out. For a linear circular graph the two give different results which is mildly annoying because this screws the distance restraint algorithm.